src/features/genderPredictor/genderPredictorRegister.js

```diff
@@ -0,0 +1,14 @@
+import {
```

Comment:
To follow the example of agc and darkMode this file should be called genderPredictor_options.js

The naming is a bit confusing because "features" and "options" have got confused along the way. I consider the flags that say whether a feature is enabled to be "options" since they are set on the options page.